extensions/azurecore/package.json

```diff
@@ -128,6 +128,11 @@
               "Microsoft Authentication Library"
             ],
             "deprecationMessage": "Warning: ADAL has been deprecated, and is scheduled to be removed in a future release. Please use MSAL (default option) instead."
+          },
+          "azure.tenantVersion": {
```

Comment:
We should either have a single global setting for ALL Azure calls (not sure if this is valid, if different calls have different valid API versions this might not work) or have a single setting but then have the configuration be a map of resource type -> API version (similar to how color customizations work)

Having a separate setting for each API call isn't really scalable and makes it more annoying to maintain.